py3status/modules/window_title.py

```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 """
-Display the current window title.
+Display window title.
```

Comment:
This needs improving maybe just revert to old one - Also module could do with a description.